src/js/accordian-input.js

```diff
@@ -0,0 +1,78 @@
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+  HOME: 36,
+  END: 35,
+  LEFT: 37,
+  UP: 38,
+  RIGHT: 39,
+  DOWN: 40
+};
+
+function getTarget(target) {
+  const $inputTarget = $(target);
+  const collapseTarget = $inputTarget.attr('href');
+  const $target = $(collapseTarget);
+
+  return $target;
+}
+
+function toggleCollapse(target) {
+  const $target = getTarget(target);
+  $target.collapse('toggle');
+}
+
+function openCollapse(target) {
+  const $target = getTarget(target);
+  $target.collapse('show');
+}
+
+function closeCollapse(target) {
+  const $target = getTarget(target);
+  $target.collapse('hide');
+}
+
+function goToFirst(target) {
+  const $target = getTarget(target);
+  const $parent = $($target.data('parent'));
+  const $first = $parent.find('[data-toggle=collapse]').first();
+
+  $first.focus();
+}
+
+function goToLast(target) {
+  const $target = getTarget(target);
+  const $parent = $($target.data('parent'));
+  const $last = $parent.find('[data-toggle=collapse]').last();
```

Comment:
Getting the first/last accordion header could potentially be simplified with something like:
```
const $last = $target.siblings('[data-toggle="collapse"]').last();
```

Reply:
I'm working on some of the other fixes, but on this one... the collapse toggles aren't necessarily going to be siblings. In the example from the Athena docs I'm using, they're each wrapped in a card, so the `<div class="card">` elements are siblings, but not the toggles inside of them.